js/main.js

```diff
@@ -20,7 +20,86 @@ function formStore(event) {
   data.nextEntryId++;
   data.entries.unshift(entry);
   $imgSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
+
+  // renderEntry(entry);
+  $ul.prepend(renderEntry(entry));
+  viewSwap('entries');
+  toggleNoEntries(data.entries.length);
```

Comment:
No argument here when you call `toggleNoEntries`. You don't have any parameters listed in your function definition, so you shouldn't be passing anything when you call it